### PR TITLE
Change license headers to standard format

### PR DESCRIPTION
**Pull request by anba**
The helper script for #384 identified some other files with non-standard or missing license headers.

### CODE REVIEW

File: test/built-ins/Math/clz32/Math.clz32_2.js [hunk 1]
```diff
@@ -1,8 +1,5 @@
...
-// 3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
-// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+// Copyright (c) 2014, Ryan Lewis. All rights reserved.
```
Comment (leobalter) [inline test/built-ins/Math/clz32/Math.clz32_2.js]: the changes can also apply to set the use of `,` (or not).

Comment (anba) [inline test/built-ins/Math/clz32/Math.clz32_2.js]: Thanks for noticing! I've pushed another commit to remove the comma. The first line in all comment headers now matches the following regexp:
`/^// Copyright (?:\([Cc]\) )?\d{4}(?:-\d{4})? [\p{IsAlphabetic}0-9 ,]+\. +All rights reserved\./`

**Comment by bterlson** [comment]
Looked it over quickly just to make sure the copyright attribution didn't change in substance. Seems good. Can we squash?

**Comment by anba** [comment]
> Can we squash?

All done!

**Comment by bterlson** [comment]
You're the best!

**Comment by anba** [comment]
Thank you!

**Comment by leobalter** [comment]
:+1:

**Pull request merged**